homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/about.css

```diff
@@ -0,0 +1,234 @@
+/*header*/
```

Comment:
Please move 
- css files to `GamesProject/styles` folder
- html files to `GamesProject` folder

Delete folder `2-css-ryvik13`cause all your work should be placed in the `GamesProject` folder

Reply:
fixed